pandas/plotting/_matplotlib/hist.py

```diff
@@ -58,13 +60,15 @@ def __init__(
         bottom: int | np.ndarray = 0,
         *,
         range=None,
+        weights=None,
```

Comment:
This is not available publicly correct i.e. doesn't need a docstring update?

Reply:
this can be passed by the user; im not inclined to update the docstring since i dont know what is accepted as it is passed through to matplotlib